internal/nvmeof/nvmeof_initiator.go

```diff
@@ -152,3 +231,45 @@ func (ni *nvmeInitiator) GetNamespaceDeviceByUUID(ctx context.Context, uuid stri
 		retry.Attempts(4), // defaults to 10 delays, too many
 	)
 }
+
+// listSubsystems retrieves current NVMe subsystem connections.
+func listSubsystems(ctx context.Context) (nvmeHostConnections, error) {
+	stdout, _, err := util.ExecCommandWithTimeout(ctx, 5*time.Second, "nvme", "list-subsys", "-o", "json")
```

Comment:
is 5 seconds enough when we have more sub system or having some network problem etc, lets keep this at 1 min for now?